amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2092,6 +2097,417 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ## Hold containers
+  '/hold-containers/{holdContainerId}/holds':
+    get:
+      tags:
+        - hold-containers
+      summary: Get holds
+      description: |
+        Returns a list of holds.
+
+        Minimal information for each child is returned by default.
+
+        You can use the **include** parameter (include=allowableOperations) to return additional information.
+      operationId: getHolds
+      parameters:
+        - $ref: '#/parameters/holdContainerIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/holdContainerEntryIncludeParam'
+        - $ref: '#/parameters/holdContainerIncludeSourceParam'
```

Comment:
I don't understand what either of these include params do

Reply:
- skipCount - The number of entities that exist in the collection before those included in this list.
- maxItems - The maximum number of items to return in the list.
- holdContainerEntryIncludeParam - Returns additional information about the hold. Any optional field from the response model can be requested. For example: path or aspectNames.
- holdContainerIncludeSourceParam - includes source (in addition to entries) with folder information on the hold container

Reply:
Ok - so if we get rid of hold containers then we can also get rid of `holdContainerIncludeSourceParam`.

I'm not convinced by loading the path or aspect names for a hold yet. I would leave out `holdContainerEntryIncludeParam` from the first version too.